data/help/texts.json

```diff
@@ -305,19 +305,19 @@
+      "Q: How can I prevent monsters from attacking me while I sleep?\nA: Find a safe place to sleep; for example, in a cleared building far from the front door.  Set traps if you have them, or build a fire to scare off wild animals.",
       "Q: Why do I always sink when I try to swim?\nA: Your swimming ability is reduced greatly by the weight you are carrying, and is also adversely affected by the clothing you wear.  Until you reach a high level of the swimming skill, you'll need to drop your equipment and remove your clothing to swim, making it a last-ditch escape plan.  Diving gear may significantly help you in swimming and diving.",
       "Q: How can I cure a fungal infection?\nA: Royal jelly, the Blood Filter bionic, and some antifungal chemicals can cure fungal infection.  You can find royal jelly in the bee hives which dot forests.  Antifungal chemicals to cure the fungal infection can either be found as random loot or made from other ingredients.",
       "Q: How do I get into science labs?\nA: You can enter the front door if you have an ID card by examining (<press_examine>) the keypad.  If you are skilled in computers and have an electrohack, it is possible to hack the keypad.  An EMP blast has a chance to force the doors open, but it's more likely to break them.  You can also sneak in through the sewers sometimes, or try to smash through the walls with explosions.",
       "Q: Why does my crafting fail so often?\nA: Check the difficulty of the recipe, and the primary skill used; your skill level should be around one and a half times the difficulty to be confident that it will succeed.",
       "Q: Why can't I carry anything?\nA: At the start of the game you only have the space in your pockets.  A good first goal of many survivors is to find a backpack or pouch to store things in.",
-      "Q: Help!  I started a fire and now my house is burning down!\nA: Fires will spread to nearby flammable tiles if they are able.  Lighting a fire in a set-up brazier, stove, wood stove, stone fireplace, pit, or even in a kitchen stove will stop it from spreading.  Fire extinguishers can put out fires that get out of control.  You can put out a fire in the fireplace by examining it.",
+      "Q: Help!  I started a fire and now my house is burning down!\nA: Fires will spread to nearby flammable tiles if they are able.  Lighting a fire in a deployed brazier, stove, wood stove, stone fireplace, pit, or even in a kitchen stove will stop it from spreading.  Fire extinguishers can put out fires that get out of control.  You can put out a fire in the fireplace by examining it.",
       "Q: I'm cold and can't sleep at night!\nA: Gather some clothes and put them in the place you use to sleep in.  Being hungry, thirsty, wet, or injured can also make you feel the cold more, so try to avoid these effects before you go to sleep.  Pillows and some other appliances will help you sleep, and the quality of your sleeping spot is also very important.",
```

Comment:
```suggestion
      "Q: I'm cold and can't sleep at night!\nA: Gather some clothes and put them in the place you sleep.  Being hungry, thirsty, wet, or injured can also make you feel the cold more, so try to avoid these effects before you go to sleep.  Pillows and some other appliances will help you sleep, and the quality of your sleeping spot is also very important.",
```

Reply:
Manual commit to "place where you're trying to sleep."